db/migrations/20210517100921_create_community_table.ts

```diff
@@ -0,0 +1,16 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable("community", (table) => {
+    table.increments().primary();
+    table.string("name").notNullable();
+    table.string("about").defaultTo(null);
```

Comment:
Consider using `table.text("about")`. Else the column is of type `VARCHAR(255)`, which can only hold 255 characters...

Reply:
Thanks, will do.